code/datums/abnormality/datum/abnormality.dm

```diff
@@ -60,6 +60,10 @@
 	var/list/transferable_var
 	///if the abno spawns with a slime radio or not
 	var/abno_radio = FALSE
+	/// Abnormality name override
+	var/name_override
+	/// Abnormality risk level override
+	var/threat_level_override
```

Comment:
Is it not possible to move displayed name/threat to a proc? I believe it would make more sense.
**If not** - change variable's name and comment so that it is possibly to tell what exactly it is overriding and where.

Reply:
Not to my knowledge, but I can update the comments.